### PR TITLE
Fix editorial issues with Terminology section.

### PR DESCRIPTION
**Pull request by msporny**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/619.html" title="Last updated on Feb 12, 2021, 5:26 PM UTC (a1c3018)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/619/5d0cdbc...a1c3018.html" title="Last updated on Feb 12, 2021, 5:26 PM UTC (a1c3018)">Diff</a>

### CODE REVIEW

File: terms.html [hunk 1]
```diff
@@ -130,10 +132,10 @@
   <dt><dfn>DID resolution</dfn></dt>
 
   <dd>
-The function that takes as its input a <a>DID</a> and a set of input
-metadata and returns a <a>DID document</a> in a conforming <a>representation</a>
+The process that takes as its input a <a>DID</a> and a set of resolution
+options and returns a <a>DID document</a> in a conforming <a>representation</a>
 plus additional metadata. This function relies on the "Read" operation of the
```
Comment (talltree) [inline terms.html:137]: Why is the word "function" left in this middle sentence?

File: terms.html [hunk 2]
```diff
@@ -149,25 +151,33 @@
...
   <dd>
 The formal syntax of a <a>decentralized identifier</a>. The generic DID scheme
-begins with the prefix <code>did:</code> as defined in <a href="#did-syntax"></a>.
-Each <a>DID method</a> specification must define a specific DID
-scheme that works with that specific <a>DID method</a>. In a specific DID method
-scheme, the DID method name must follow the first colon and terminate with the
-second colon, e.g., <code>did:example:</code>
+begins with the prefix <code>did:</code> as defined in <a
+href="#did-syntax"></a>. Each <a>DID method</a> specification defines a specific
+DID scheme that works with that specific <a>DID method</a>. In a specific DID
```
Comment (talltree) [inline terms.html:156]:
```suggestion
DID method scheme that works with that specific <a>DID method</a>. In a specific DID
```

**Review by talltree — APPROVED** [review]

File: terms.html [hunk 3]
```diff
@@ -50,46 +50,48 @@
...
+enabling the child to <a>authenticate</a> using the DID. However, the child
+might not be permitted to add other personal devices without the parent's
+permission.
   </dd>
 
   <dt><dfn data-lt="DID documents">DID document</dfn></dt>
 
   <dd>
 A set of data describing the <a>DID subject</a>, including mechanisms, such as
-public keys and pseudonymous biometrics, that the <a>DID subject</a> or a
```
Comment (peacekeeper) [inline terms.html:85]: Agree with removing "pseudonymous biometrics" here. If we wanted to say anything about biometrics, we should have done that elsewhere, not in the terminology section.

File: terms.html [hunk 4]
```diff
@@ -149,25 +151,33 @@
...
   <dd>
-The entity identified by a <a>DID</a> and described by a <a>DID document</a>. A
-<a>DID</a> has exactly one DID subject. Anything can be a DID subject: person,
-group, organization, physical thing, digital thing, logical thing, etc.
+The entity identified by a <a>DID</a> and described by a <a>DID document</a>.
+Anything can be a DID subject: person, group, organization, physical thing,
+digital thing, logical thing, etc.
+  </dd>
+
+  <dt><dfn data-lt="service">DID services</dfn></dt>
```
Comment (peacekeeper) [inline terms.html:169]: I'd rather not change this from "service" to "DID services", to be consistent with the section https://w3c.github.io/did-core/#services. Just like we also say "verification methods", not "DID verification methods".

File: terms.html [hunk 5]
```diff
@@ -177,17 +187,16 @@
...
-resource might be a <a>DID document</a> plus additional metadata, or it might be a
-secondary resource contained within the <a>DID document</a>, or it might be a
-resource entirely external to the <a>DID document</a>. The function
-uses the <a>DID resolution</a> function to fetch a
-<a>DID document</a> indicated by the <a>DID</a> contained within the
-<a>DID URL</a>. The dereferencing function then can perform additional
-processing on the <a>DID document</a> to return the dereferenced resource
-indicated by the <a>DID URL</a>. The inputs and outputs of this function are
-defined in <a href="#did-url-dereferencing"></a>.
+The function that takes as its input a <a>DID URL</a> and a set of input
```
Comment (peacekeeper) [inline terms.html:190]: Do you also want to change "function" to "process" here, to be consistent with your changes in "DID resolution". I'm fine with either term, but it should be the same for both term definitions.

**Review by peacekeeper — CHANGES_REQUESTED** [review]

File: terms.html (hunk 1 above)
Comment (TallTed) [inline terms.html:137]:
```suggestion
plus additional metadata. This process relies on the "Read" operation of the
```

File: terms.html [hunk 6]
```diff
@@ -102,21 +104,21 @@
...
-A definition of how a specific <a>DID scheme</a> must be implemented to work
-with a specific <a>verifiable data registry</a>. A DID method is defined by a
-DID method specification, which must specify the precise operations by which
-<a>DIDs</a> are created, resolved and deactivated and <a>DID documents</a> are
-written and updated. See <a href="#methods"></a>.
+A definition of how a specific <a>DID scheme</a> is implemented to work with a
+specific <a>verifiable data registry</a>. A DID method is defined by a DID
+method specification, which must specify the precise operations by which
+<a>DIDs</a> and <a>DID documents</a> are created, resolved, updated, and
+deactivated. See <a href="#methods"></a>.
```
Comment (brentzundel) [inline terms.html:111]:
```suggestion
A definition of how a specific <a>DID scheme</a> is implemented. A DID method is
defined by a DID method specification, which must specify the precise operations by
which <a>DIDs</a> and <a>DID documents</a> are created, resolved, updated,
and deactivated. See <a href="#methods"></a>.
```

File: terms.html [hunk 7]
```diff
@@ -50,46 +50,48 @@
...
-<a>representations</a> as defined in <a href="#representations"></a> or in
-the W3C DID Specification Registries [[DID-SPEC-REGISTRIES]].
+cryptographic public keys, that the <a>DID subject</a> or a <a>DID delegate</a>
+can use to <a>authenticate</a> itself and prove its association with the
+<a>DID</a>. A DID document might also contain other <a
+href="https://en.wikipedia.org/wiki/Attribute_(computing)">attributes</a> or <a
+href="https://en.wikipedia.org/wiki/Claims-based_identity">claims</a> describing
+the <a>DID subject</a>. A DID document might have one or more different
+<a>representations</a> as defined in <a href="#representations"></a> or in the
+W3C DID Specification Registries [[DID-SPEC-REGISTRIES]].
```
Comment (brentzundel) [inline terms.html:94]:
```suggestion
<a>DID</a>. A DID document might have one or more different
<a>representations</a> as defined in <a href="#representations"></a> or in the
W3C DID Specification Registries [[DID-SPEC-REGISTRIES]].
```

File: terms.html [hunk 8]
```diff
@@ -102,21 +104,21 @@
...
   <dt><dfn data-lt="DID paths">DID path</dfn></dt>
 
   <dd>
 The portion of a <a>DID URL</a> that begins with and includes the first forward
-slash (<code>/</code>) character and ends with either a question mark (<code>?</code>)
-character or a fragment hash sign (<code>#</code>) character (or the end of the
-DID URL). DID path syntax is identical to URI path syntax. See
-<a href="#path"></a>.
+slash (<code>/</code>) character and ends with either a question mark
+(<code>?</code>) character, or a fragment hash sign (<code>#</code>) character,
```
Comment (brentzundel) [inline terms.html:119]:
```suggestion
(<code>?</code>) character, a fragment hash sign (<code>#</code>) character,
```

File: terms.html [hunk 9]
```diff
@@ -149,25 +151,33 @@
...
+Anything can be a DID subject: person, group, organization, physical thing,
+digital thing, logical thing, etc.
+  </dd>
+
+  <dt><dfn data-lt="service">DID services</dfn></dt>
+  <dd>
+Means of communicating or interacting with the <a>DID subject</a> or
+associated entities via one or more <a>service endpoints</a>.
+Examples include discovery services, agent services, social networking
+services, file storage services, and verifiable credential repository services.
```
Comment (brentzundel) [inline terms.html:174]:
```suggestion
Examples of services include discovery, agent, social networking, file storage,
and verifiable credential repository services.
```

File: terms.html (hunk 5 above)
Comment (brentzundel) [inline terms.html:190]:
```suggestion
The process that takes as its input a <a>DID URL</a> and a set of input
```

File: terms.html [hunk 10]
```diff
@@ -177,17 +187,16 @@
...
+The function that takes as its input a <a>DID URL</a> and a set of input
+metadata, and returns a <a>resource</a>. This resource might be a <a>DID
+document</a> plus additional metadata, or it might be a secondary resource
+contained within the <a>DID document</a>, or it might be a resource entirely
+external to the <a>DID document</a>. The function uses the <a>DID resolution</a>
+function to fetch a <a>DID document</a> indicated by the <a>DID</a> contained
+within the <a>DID URL</a>. The dereferencing function then can perform
+additional processing on the <a>DID document</a> to return the dereferenced
+resource indicated by the <a>DID URL</a>. The inputs and outputs of this
+function are defined in <a href="#did-url-dereferencing"></a>.
```
Comment (brentzundel) [inline terms.html:199]:
```suggestion
external to the <a>DID document</a>. The process uses <a>DID resolution</a> to
fetch a <a>DID document</a> indicated by the <a>DID</a> contained within the
<a>DID URL</a>. The dereferencing process can then perform additional processing
on the <a>DID document</a> to return the dereferenced resource indicated by the
<a>DID URL</a>. The inputs and outputs of this process are defined in
<a href="#did-url-dereferencing"></a>.
```

**Review by brentzundel — APPROVED** [review]
I've suggested some re-wordings that I feel add to the readability of the text. Feel free to accept or reject them as you prefer.

File: terms.html [hunk 11]
```diff
@@ -50,46 +50,48 @@
...
 
   <dd>
-An entity that has the capability to make changes to a <a>DID document</a>.
-A <a>DID</a> might have more than one DID controller. The <a>DID controller(s)</a>
-can be denoted by the optional `controller` property at the top level of the <a>DID
-document</a>. Note that one <a>DID controller</a> might be the <a>DID subject</a>.
+An entity that has the capability to make changes to a <a>DID document</a>. A
+<a>DID</a> might have more than one DID controller. The <a>DID controller(s)</a>
+can be denoted by the optional `controller` property at the top level of the
+<a>DID document</a>. Note that a <a>DID controller</a> might be the <a>DID
```
Comment (rhiaro) [inline terms.html:65]:
```suggestion
<a>DID</a> might have more than one DID controller. The DID controller(s)
can be denoted by the optional `controller` property at the top level of the
<a>DID document</a>. Note that a DID controller might be the <a>DID
```

remove self-referencing definition

File: terms.html [hunk 12]
```diff
@@ -50,46 +50,48 @@
...
-would contain the private cryptographic material enabling the child to
-<a>authenticate</a> using the DID. However the child might not be permitted to
-add other personal devices without the parent's permission.
+An entity to whom a <a>DID controller</a> has granted permission to use a
+<a>verification method</a> associated with a <a>DID</a> via a <a>DID
+document</a>. For example, a parent who controls a child's <a>DID document</a>
+might permit the child to use their personal device in order to
+<a>authenticate</a>. In this case, the child is the <a>DID delegate</a>. The
+child's personal device would contain the private cryptographic material
+enabling the child to <a>authenticate</a> using the DID. However, the child
```
Comment (rhiaro) [inline terms.html:78]:
```suggestion
enabling the child to <a>authenticate</a> using the <a>DID</a>. However, the child
```

File: terms.html [hunk 13]
```diff
@@ -102,21 +104,21 @@
...
 
   <dd>
 The portion of a <a>DID URL</a> that begins with and includes the first forward
-slash (<code>/</code>) character and ends with either a question mark (<code>?</code>)
-character or a fragment hash sign (<code>#</code>) character (or the end of the
-DID URL). DID path syntax is identical to URI path syntax. See
-<a href="#path"></a>.
+slash (<code>/</code>) character and ends with either a question mark
+(<code>?</code>) character, or a fragment hash sign (<code>#</code>) character,
+or the end of the DID URL. DID path syntax is identical to URI path syntax.
```
Comment (rhiaro) [inline terms.html:120]:
```suggestion
or the end of the <a>DID URL</a>. DID path syntax is identical to URI path syntax.
```

File: terms.html (hunk 9 above)
Comment (rhiaro) [reply terms.html:174]: Could also include "negotiator", "mediator", and/or "proxy" as examples of services, to correspond with recent updates in the Service Privacy section

**Review by rhiaro — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**